chapter_introduction/index_vn.md

```diff
@@ -1936,7 +1949,8 @@ or to *explore* the space of strategies,
 potentially giving up some short-run reward in exchange for knowledge.
 -->
 
-*dịch đoạn phía trên*
+Cuối cùng, tại 1 thời điểm cho trước, những người học tăng cường sẽ biết đến 1 cách giải quyết tốt, những sẽ có những cách giải quyết khác tốt hơn mà họ chưa bao giờ thử. 
+Những người học tăng cường phải luôn luôn lựa chọn việc *khai phá* giải pháp tốt nhất được biết đến hiện tại, hay *khám phá* thêm những giải pháp khác, từ bỏ một số phần thưởng ngắn hạn để đổi lại kiến thức. 
```

Comment:
từ `khai phá` hay quá!

Reply:
"Khai phá" và "khám phá" trong tiếng Việt có nghĩa gần như nhau.

"Khai phá": Biến đất hoang thành ruộng nương bằng sức lao động.
"Khám phá": tìm ra, phát hiện ra cái còn ẩn giấu, cái bí mật.

Trong khi đó, theo từ điển Google:
"exploit": make full use of and derive benefit from (a resource).
"explore": travel in or through (an unfamiliar country or area) in order to learn about or familiarize oneself with it.

Theo nghĩa tiếng Anh thì "explore" là đi tìm cái mới, "exploit" là tiếp tục tận dụng cái cũ. Tiếng Việt cũng dùng thừ "khai thác" có nghĩa tương đương với "exploit". Mình vote từ này dịch là "khai thác" hơn là "khai phá".

Ở một khía cạnh khác, "khai phá" được dịch từ "mining" trong "khai phá dữ liệu" (data mining) nên mình tránh dùng lại một từ trong nhiều văn cảnh khác nhau.